src/beans/Course.js

```diff
@@ -82,28 +81,24 @@ class Course {
   }
 
   async fetchGpa() {
-    const url = `https://critique.gatech.edu/course?courseID=${encodeURIComponent(
-      this.id
-    )}`;
+    const base =
+      'https://c4citk6s9k.execute-api.us-east-1.amazonaws.com/test/data';
+    const encodedCourse = encodeURIComponent(this.id);
     return axios({
-      url: `https://cors-anywhere.herokuapp.com/${url}`,
-      method: 'get',
-      headers: {
-        'X-Requested-With': 'XMLHttpRequest',
-        'Content-Type': 'text/html'
-      }
+      url: `${base}/course?courseID=${encodedCourse}`,
+      method: 'get'
     }).then((response) => {
```

Comment:
We should make sure to handle a possible promise rejection (via `.catch(...)`) in case the request fails

Reply:
Alternatively, since this is an `async` function, you can use [async/await syntax](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await) and a [try...catch block](https://itnext.io/error-handling-with-async-await-in-js-26c3f20bc06a).